packages/dm-core-plugins/src/generic-list/GenericTablePlugin.tsx

```diff
@@ -94,96 +110,88 @@ export const GenericTablePlugin = (
+        <Table style={{ width: '100%' }}>
+          <Table.Head>
+            <Table.Row>
+              {internalConfig.columns.map((attribute: string) => (
+                <Table.Cell key={attribute}>{attribute}</Table.Cell>
+              ))}
+              {internalConfig.editMode && internalConfig.showDelete && (
+                <Table.Cell>Delete</Table.Cell>
+              )}
+            </Table.Row>
+          </Table.Head>
+          <Table.Body>
+            {Object.entries(items).map(([key, item], index) => (
+              <Table.Row key={key}>
+                {internalConfig.columns.map((attribute: string) => {
+                  if (typeof item[attribute] === 'object')
+                    throw new Error(
+                      `Objects can not be displayed in table. Attribute '${attribute}' is not a primitive type.`
+                    )
+                  // TODO: Consider having a more robust way of getting type and validating form
+                  const attributeType = typeof item[attribute]
+                  return (
+                    <Table.Cell key={attribute}>
+                      <Input
+                        value={item[attribute] ?? ''}
+                        readOnly={
+                          attribute === 'type' || !internalConfig.editMode
+                        }
+                        type={attributeType}
+                        rightAdornments={
+                          <>{internalConfig.editMode ? attributeType : ''}</>
+                        }
+                        onChange={(event: ChangeEvent<HTMLInputElement>) => {
+                          let newValue: string | number | boolean =
+                            event.target.value
+                          if (attributeType === 'number')
+                            newValue = Number(newValue)
+                          updateItem(key, attribute, newValue)
+                        }}
+                      />
+                    </Table.Cell>
+                  )
+                })}
 
-        <Button aria-disabled={isSaveLoading} onClick={() => saveTable()}>
-          {isSaveLoading ? <Progress.Dots color={'primary'} /> : 'Save'}
-        </Button>
+                {internalConfig.editMode && internalConfig?.showDelete && (
+                  <Table.Cell style={{ textAlign: 'right' }}>
+                    <Button
+                      variant="ghost_icon"
+                      color="danger"
+                      onClick={() => deleteItem(`${idReference}.${index}`, key)}
+                    >
+                      <Icon data={delete_to_trash} title="Delete" />
+                    </Button>
+                  </Table.Cell>
+                )}
+              </Table.Row>
+            ))}
+          </Table.Body>
+        </Table>
+        <div>
+          {internalConfig.editMode && (
+            <Button
+              aria-disabled={isSaveLoading || !dirtyState}
```

Comment:
aria-disabled doesn't actually disable the onclick, so if you try clicking the button it will fire the onclick event.
![image](https://user-images.githubusercontent.com/64828956/232714334-3e01c0b9-880e-4578-a7c7-6fbd21a4e560.png)
Instead do:
```suggestion
              disabled={isSaveLoading || !dirtyState}
```

Reply:
fixed